lib/web_ui/dev/build.dart

```diff
@@ -45,6 +46,12 @@ class BuildCommand extends Command<bool> with ArgUtils<bool> {
           'output will be located at "out/wasm_profile".\nThis only applies to '
           'the wasm build. The host build is always built in release mode.',
     );
+    argParser.addFlag(
```

Comment:
Do we have logic to barf if both `debug` and `profile` are set?

Reply:
You're right, I should probably add that.

Reply:
Nah, you were right, they are mutually exclusive. It should just barf.